floris/simulation/grid.py

```diff
@@ -265,6 +276,33 @@ def set_grid(self) -> None:
         self.y = np.take_along_axis(self.y_sorted, self.unsorted_indices, axis=2)
         self.z = np.take_along_axis(self.z_sorted, self.unsorted_indices, axis=2)
 
+        # Now calculate grid coordinates in original frame (from 270 deg perspective)
+        self.x_inertial_frame, self.y_inertial_frame, self.z_inertial_frame = \
```

Comment:
The `TurbineGrid` contains points only on the rotor planes, so this is not used for the horizontal slice visualizations. I think these are not required here since they would not be used.

Reply:
I think you are right. See also my comment above. I think the main variables that we need to properly apply heterogeneous inflow conditions are `grid.x_sorted_inertial_frame`, and `grid.y_sorted_inertial_frame`.

Reply:
I removed the code not needed in this commit (it also contains some small formatting changes to allow me to commit).

https://github.com/NREL/floris/pull/578/commits/624ca2e489a6fb91fa554a0b0592e9aced77064a